decidim-core/app/commands/decidim/create_report.rb

```diff
@@ -74,6 +75,19 @@ def send_report_notification_to_moderators
       ReportedMailer.send_report_notification_to_users(participatory_space_moderators, @report)
     end
 
+    def send_report_notification_to_author
+      data = {
+        event: "decidim.events.reports.report_created",
+        event_class: Decidim::ReportCreatedEvent,
+        resource: @report.moderation.reportable,
+        extra: {
+          report_reason: @report.reason
+        },
+        affected_users: @report.moderation.reportable.try(:authors) || [@report.moderation.reportable.try(:author)]
```

Comment:
I'm not sure if it's better to use `normalized_author` instead of `author` 🤔

Reply:
Yup, seems like you're right! Nice catch, thanks!